# GarciaArocaMycologia2020

The alignments for the maximum likelihood (ML) and Bayesian phylogenetic analyses as well as the resulting inferred phylogenies are contained in this repository. The input files and resulting trees are contained in folders for the ML and Bayesian analyses, resepctively.

Below is information regarding methods pertinent to the data in this repository. These methods are an excerpt from the manuscript submitted to Mycologia.

Title: Xylaria necrophora sp. nov. is an emerging root-associated pathogen responsible for taproot decline of soybean in the southern United States

Authors: Teddy Garcia-Aroca<sup>a</sup>, Paul P Price<sup>b</sup>, Brooklyn M. Squiers<sup>a</sup>, Maria Tomaso-Peterson<sup>c</sup>, Tessie Wilkerson<sup>d</sup>, Terry N. Spurlock<sup>e</sup> Travis Faske<sup>f</sup>, Burt Bluhm<sup>g</sup>, Kassie Conner<sup>h</sup>, Edward Sikora<sup>h</sup>, Rachel Guyer<sup>i</sup>, Heather Kelly<sup>i</sup>; Tom Allen<sup>j</sup>, and Vinson P. Doyle<sup>a,*</sup>

<sup>*</sup> Correspondence to: Vinson P. Doyle, Department of Plant Pathology & Crop Physiology, Louisiana State University, Baton Rouge, LA 70803. E-mail: VDoyle@agcenter.lsu.edu 

<sup>a</sup> Department of Plant Pathology & Crop Physiology, Louisiana State University, Baton Rouge, LA; 

<sup>b</sup> LSU AgCenter, Macon Ridge Research Station, Winnsboro, LA;

<sup>c</sup> Department of Biochemistry, Molecular Biology, Entomology, & Plant Pathology, Mississippi State University, Mississippi State, MS;

<sup>d</sup> Department of Biochemistry, Molecular Biology, Entomology, & Plant Pathology, Mississippi State University, Stoneville, MS;

<sup>e</sup> Department of Plant Pathology, University of Arkansas System Division of Agriculture Cooperative Extension Service, Monticello, AR;

<sup>f</sup> Department of Plant Pathology, University of Arkansas System Division of Agriculture Cooperative Extension Service, Lonoke, AR;

<sup>g</sup> Department of Plant Pathology, University of Arkansas, Fayetteville, AR;

<sup>h</sup> Alabama Cooperative Extension System, Auburn University, Auburn, AL;

<sup>i</sup> Department of Entomology and Plant Pathology, West Tennessee Research & Education Center, University of Tennessee, Jackson, TN;

<sup>i</sup> Delta Research and Extension Center, Mississippi State University, Stoneville, MS;



## Methods (Phylogeny-related)

### Sampling of fungal isolates from Taproot decline (TRD) and historical specimens

Soybean plants that exhibited symptoms of TRD as described by Allen et al. (2017), such as interveinal chlorosis followed by necrosis on the foliage and localized necrosis in the root system with mycelial growth in the root pith near the base of the stem, were collected from soybean farms across the region where the disease has been reported or observed. Samples were randomly collected across the southern United States, aiming to collect representative specimens from the known distribution of the disease based on recent unofficial reports, including Alabama, Arkansas, Louisiana, Mississippi, and Tennessee. Plants were removed from the ground by digging around the estimated area where the root system was expected to appear and pulling the plant off the ground to conserve the integrity of the roots. Samples were stored in plastic, zip-loc bags in a cooler and transported to the laboratory for isolation of the pathogen. Roots and stems were initially rinsed with water to remove any remaining soil and debris, then surface sterilized by submerging in a 0.6% sodium hypochlorite solution for 10 minutes and rinsed with distilled water. Root sections with fungal stroma on the surface were dissected to smaller pieces and submerged in a 0.6% sodium hypochlorite solution for 2 minutes using a strainer. Sterilized root fragments were then placed on potato dextrose agar (PDA) media amended with streptomycin (125 mg/L) and chloramphenicol (75 mg/L) (PDA-CS). Plates were incubated in the dark at 20 C for 2-4 weeks until mycelial development of Xylaria sp. was observed. When mycelia could be observed in the pith of diseased plants, a sterilized needle was used to transfer mycelia straight to PDA-CS.

Sampling of historical specimens: we obtained historical specimens deposited in US herbaria that were classified as X. arbuscula based on morphological features. We sampled 28 specimens collected in the 1920’s, 1930’s, and 1940’s and deposited in the U.S. National Fungus Collection (BPI) in addition to five specimens collected in the 1990’s in the Islands of Hawaii and deposited in Jack D. Rogers (Professor Emeritus, Washington State University) personal herbarium (JDR specimens). Specimens deposited as X. arbuscula in US herbaria were sampled because the distribution of X. arbuscula in north America overlaps with that of the TRD pathogen and was shown to be closely related to the TRD pathogen in a previous study (Allen et al. 2017).

### DNA extraction

Isolates representative of the geographic distribution of our collections were used to study the phylogenetic diversity and placement of Xylaria sp. associated with TRD. Total genomic DNA was extracted from mycelia of pure cultures using a modified protocol based on the Doyle & Doyle (1987) method for extraction of nucleic acids. Fungal tissue was scrapped from plates using a sterilized scalpel and deposited in a 1.5 mL Eppendorf safe-lock tube containing zirconium beads (~1g of 2.0 mm and ~1g of 0.5 mm beads) and 400 µL of CTAB (cetyltrimethylammonium bromide) buffer (0.1 M). Safe-lock tubes were placed in a bullet blender (Next Advance, New York, USA) for 3 min at a speed of 7000 rpm to grind tissue and release genomic DNA into the CTAB buffer before adding 380 µL of CTAB and 20 µL of proteinase K. The lysate was incubated at 55 oC in a heat block for 1 hour with vortexing every 15 min. Tubes were centrifuged for 5 min at maximum speed (14000 rpm/20000 rcf) and the supernatant was transferred to 2.0 mL tubes containing 800 µL of CIA [chloroform:isoamyl alcohol (24:1)], vortexed for two seconds and centrifuged for 20 min. The upper phase (~700 uL) was transferred to a new 2.0 mL tube containing 700 µL of CIA, vortexed for a few seconds and centrifuged for 10 min. The upper phase (~600 mL) was transferred to 1.5 mL tubes containing 600 mL of isopropanol, incubated at -20 oC for at least 30 min, and centrifuged for 5 min at maximum speed (~14000 rpm/20000 rcf). The supernatant was decanted and the remaining pellet was washed with 70% cold ethanol, mixed by inversion and centrifuged for 1 min. Ethanol was decanted and tubes were centrifuged again for 30 seconds. Excess ethanol was removed initially by pipetting, then air drying for 10 min. Finally, a Tris-EDTA (TE) buffer and 1% RNase solution was used to re-suspend the DNA by incubating the tubes for 1 hour at 37 C and at 65 C for 30 min. DNA of dried specimens from US herbaria was isolated using a Qiagen DNeasy® Plant Mini Kit following the manufacturer protocol. Genomic DNA was diluted in nuclease free water to a working concentration of 10-20 ng/µL when undiluted template DNA did not amplify in PCR reactions.
 


### Data assembly for Phylogenetic analysis of Multi-locus DNA sequences

We aligned sequence data generated in this study with data from Hsieh et al. (2010), U’Ren et al. (2016), Amnuaykanjanasin et al. (2005), Robinson and Natvig (2019), Maryono et al. (2019), and Allen et al. (2017) for a total of 138 nrITS, 132 ACT, 132 TUB2, and 132 RPB2 sequences from these studies (Supplementary Table 1). The complete dataset included published sequences from 138 specimens/isolates (Supplementary Table 2), along with sequences of representative TRD-associated isolates from the region, sequences of historical specimens found in the region as saprophytes or endophytes in agricultural and non-agricultural habitats, contemporary specimens found in the region in non-agricultural habitats as saprophytes or endophytes, and specimens collected outside of the region (Table 1). Sequences of ACT, TUB2, and RPB2 from a specimen collected in China on wild rice (Xylaria striata RK1-1) were sliced from the genome (Genbank whole genome shotgun project LOBO00000000.1) (Robinson and Natvig 2019) and included in analyses. The total number of taxa per locus (published sequences and newly added sequences from this study) was 176 for nrITS, 160 for ACT, 158 for RPB2, and 156 TUB2. Sequences of each locus were aligned using the online version of MAFFT with the GINS-i parameters, a 200PAM/k=2 scoring matrix, and a gap opening penalty of 1.53 (Katoh, Kuma, Toh, & Miyata, 2005; Katoh et al., 2002). Uncertain sites in the alignment were identified using Guidance 2 v. 2.02 (Landan and Graur 2008, Penn et al. 2010, Privman et al. 2012, Sela et al. 2015) by assigning all residues in the alignment a certainty score. The default Guidance 2 v. 2.02 parameters were applied to the unaligned sequences using MAFFT as the alignment program, nucleotides as the sequence type, 100 bootstrap iterations, column score cutoff as 0.93, and sequence score cutoff as 0.6. Residues below a particular score were then masked by replacing the respective nucleotide with an 'N’, using the maskLowScoreResidues.pl script. This type of filtering was chosen because it retains the most phylogenetically informative sites compared to other filtering algorithms, such as Gblocks (Castresana 2000, Talavera and Castresana 2007, Privman et al. 2012, Vieira et al. 2017). The optimal filtering cutoff for individual residues was chosen by examining the impact of filtering on bootstrap support and monophyly of the focal lineage in preliminary analyses. Based on these empirical assessments, a filtering cutoff of 0.07 was used for the nrITS and 0.2 for ACT, while the default cutoff of 0.93 was used for RPB2 and TUB2 (Supplementary Figure 1A-D). Individual alignments were concatenated into a multi-locus alignment in Sequence Matrix (Vaidya et al. 2011). The maximum likelihood phylogeny for each gene and the concatenated alignment were estimated using RAxML v. 8.2.12 (Stamatakis 2014) assuming the general time reversible (GTR) model plus a -distribution on rates of evolution across sites on the CIPRES/XSEDE platform (Miller et al. 2010). Support values were estimated with 1000 bootstrap pseudoreplicates for each analysis. Prior to Bayesian analyses, the best-fit model of nucleotide evolution for each alignment was estimated in jModelTest 2.1.7 (Darriba, et al., 2015; Guindon & Gascuel, 2003) using the Akaike Information Criterion (AIC) corrected for small sample sizes (AICc) (Hurvich and Tsai 1989), using 11 substitution schemes, estimating base frequencies, gamma and invariant sites for rate variation parameters, improved neighbor-joining algorithm (BIONJ) for likelihood calculations (Gascuel 1997), and the normalized mutual information (NMI) criterion (Jain, A et al. 1999, Cai and Sun 2011, Romano et al. 2014) for clustering in the base tree search. Bayesian inference was done in MrBayes v3.2.7 (Huelsenbeck and Ronquist 2001, Ronquist and Huelsenbeck 2003) assuming the best-fit model selected in jModelTest compatible with MrBayes by partition with each partition corresponding to each locus. The model parameters and topology were estimated with four replicate Markov chain Monte Carlo (MCMC) runs per analyses, four Metropolis-coupled chains per run (3 cold, one heated), and six million generations per run (Tavaré 1986, Sumner et al. 2012, Yang 2014, Arenas 2015). Convergence was determined by checking that the standard deviation of split frequencies was below 0.05 and the effective sample sizes (ESS) were above 200 after removing the first 25% of samples as burn-in. The majority-rule consensus phylogeny was computed using SumTrees v. 4.3.0 in Dendropy v. 4.4.0 (Sukumuran and Holder, 2010). Posterior probabilities equal to or greater than 0.85 were mapped, if present, to the maximum likelihood phylogeny alongside the maximum likelihood bootstrap support values greater than 70%. The individual and concatenated alignments along with estimated phylogenies were uploaded to GitHub (github repository: https://github.com/vinsondoyle/GarciaArocaMycologia2020).

## References

1. Allen T, Bluhm B, Conner K, Doyle VP, Price T, Sikora E, Singh R, Spurlock T, Tomaso-Peterson M, Wilkerson T. 2017. First Description of the Causal Agent of Taproot Decline of Soybean , an Emerging Disease in the Southern United States. 18:35–40.
2. Amnuaykanjanasin A, Punya J, Paungmoung P, Rungrod A, Tachaleat A, Pongpattanakitshote S, Cheevadhanarak S, Tanticharoen M. 2005. Diversity of type I polyketide synthase genes in the wood-decay fungus Xylaria sp. BCC 1067. FEMS Microbiol Lett. 251:125–136, doi:10.1016/j.femsle.2005.07.038.
Arenas M. 2015. Trends in substitution models of molecular evolution. Front Genet. 6, doi:10.3389/fgene.2015.00319.
3. Cai Y, Sun Y. 2011. ESPRIT-Tree: Hierarchical clustering analysis of millions of 16S rRNA pyrosequences in quasilinear computational time. Nucleic Acids Res. 39:1–10, doi:10.1093/nar/gkr349.
4. Castresana J. 2000. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 17:540–552, doi:10.1093/oxfordjournals.molbev.a026334.
5. Darriba D, Taboada GL, Doallo R, Posada D. 2015. Europe PMC Funders Group jModelTest 2 : more models , new heuristics and high- performance computing. Nat Methods. 9:6–9, doi:10.1038/nmeth.2109.jModelTest.
6. Doyle JJ, Doyle JL. 1987. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem Bull. 19:11–15.
7. Gascuel O. 1997. BIONJ: An improved version of the NJ algorithm based on a simple model of sequence data. Mol Biol Evol. 14:685–695, doi:10.1093/oxfordjournals.molbev.a025808.
Guindon S, Gascuel O. 2003. A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood. 696–704 p.
8. Hsieh HM, Lin CR, Fang MJ, Rogers JD, Fournier J, Lechat C, Ju YM. 2010. Phylogenetic status of Xylaria subgenus Pseudoxylaria among taxa of the subfamily Xylarioideae (Xylariaceae) and phylogeny of the taxa involved in the subfamily. Mol Phylogenet Evol. 54:957–969, doi:10.1016/j.ympev.2009.12.015.
9. Huelsenbeck JP, Ronquist F. 2001. MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics. 17:754–755, doi:10.1093/bioinformatics/17.8.754.
10. Hurvich CM, Tsai CL. 1989. Regression and time series model selection in small samples. Biometrika. 76:297–307, doi:10.1093/biomet/76.2.297.
11. Jain, A K, Murty, M P, Flynn, P J. 1999. Data clustering: a review. ACM Comput Surv. 31:264–323, doi:10.1145/345966.346030.
12. Katoh K, Kuma KI, Toh H, Miyata T. 2005. MAFFT version 5: Improvement in accuracy of multiple sequence alignment. Nucleic Acids Res. 33:511–518, doi:10.1093/nar/gki198.
13. Katoh K, Misawa K, Kuma K, Miyata T. 2002. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. Nucleic Acids Res. 30:3059–3066, doi:10.1093/nar/gkf436.
14. Landan G, Graur D. 2008. Local reliability measures from sets of co-optimal multiple sequence alignments. Pacific Symp Biocomput 2008, PSB 2008. 24:15–24, doi:10.1142/9789812776136_0003.
15. Maryono T, Widiastuti A, Murti RH, Priyatmojo A. 2019. Identification and Characterization of the Causal Agent of Sugarcane Root and Basal Stem Rot in South Sumatra, Indonesia. Sugar Tech. doi:10.1007/s12355-019-00749-2.
16. Miller MA, Pfeiffer W, Schwartz T. 2010. Creating the CIPRES Science Gateway for inference of large phylogenetic trees. 2010 Gatew Comput Environ Work GCE 2010. doi:10.1109/GCE.2010.5676129.
17. Penn O, Privman E, Ashkenazy H, Landan G, Graur D, Pupko T. 2010. GUIDANCE: A web server for assessing alignment confidence scores. Nucleic Acids Res. 38:23–28, doi:10.1093/nar/gkq443.
18. Privman E, Penn O, Pupko T. 2012. Improving the performance of positive selection inference by filtering unreliable alignment regions. Mol Biol Evol. 29:1–5, doi:10.1093/molbev/msr177.
19. Robinson AJ, Natvig DO. 2019. Diverse members of the Xylariales lack canonical mating-type regions. Fungal Genet Biol. 122:47–52, doi:10.1016/j.fgb.2018.12.004.
20. Romano S, Bailey J, Vinh NX, Verspoor K. 2014. Standardized mutual information for clustering comparisons: One step further in adjustment for chance. 31st Int Conf Mach Learn ICML 2014. 4:2873–2882.
21. Ronquist F, Huelsenbeck JP. 2003. MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics. 19:1572–1574, doi:10.1093/bioinformatics/btg180.
22. Sela I, Ashkenazy H, Katoh K, Pupko T. 2015. GUIDANCE2: Accurate detection of unreliable alignment regions accounting for the uncertainty of multiple parameters. Nucleic Acids Res. 43:W7–W14, doi:10.1093/nar/gkv318.
23. Stamatakis A. 2014. RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 30:1312–1313, doi:10.1093/bioinformatics/btu033.
24. Sumner JG, Jarvis PD, Fernández-Sánchez J, Kaine BT, Woodhams MD, Holland BR. 2012. Is the general time-reversible model bad for molecular phylogenetics? Syst Biol. 61:1069–1074, doi:10.1093/sysbio/sys042.
25. Talavera G, Castresana J. 2007. Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol. 56:564–577, doi:10.1080/10635150701472164.
26. Tavaré S. 1986. Some probabilistic and statistical problems in the analysis of DNA sequences. Am Math Soc Lect Math Life Sci. 17:57–86, doi:citeulike-article-id:4801403.
27. U’Ren JM, Miadlikowska J, Zimmerman NB, Lutzoni F, Stajich JE, Arnold AE. 2016. Contributions of North American endophytes to the phylogeny, ecology, and taxonomy of Xylariaceae (Sordariomycetes, Ascomycota). Mol Phylogenet Evol. 98:210–232, doi:10.1016/j.ympev.2016.02.010.
28. Vaidya G, Lohman DJ, Meier R. 2011. SequenceMatrix: Concatenation software for the fast assembly of multi-gene datasets with character set and codon information. Cladistics. 27:171–180, doi:10.1111/j.1096-0031.2010.00329.x.
29. Vieira WAS, Lima WG, Nascimento ES, Michereff SJ, Câmara MPS, Doyle VP. 2017. The impact of phenotypic and molecular data on the inference of Colletotrichum diversity associated with Musa. Mycologia. 109:912–934, doi:10.1080/00275514.2017.1418577.
30. Yang Z. 2014. Molecular Evolution: A Statistical Approach. Oxford: Oxford University Press. 492 p.



#Christine and Teddy were here
#Christine and Teddy were here twice
